.github/workflows/python-package.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       run: |
         export MSPASS_HOME=$(pwd)
         export SPARK_LOCAL_IP=127.0.0.1
-        pytest --cov=mspasspy python/tests/
+        for file in `find python/tests -type f -name "test_*.py"`; do pytest --cov=mspasspy $file; done
```

Comment:
@Aristoeu Why do you change this line?

Reply:
The tests run via pytest are run in a single process and the import will be a mess, as shown in this link:
https://stackoverflow.com/questions/55229163/pytest-independent-import-of-the-same-module-from-different-test-files

But if using `pytest --forked`, tests will be able to run in different processes and the problem can be partially solved, as shown in the link below. 
https://stackoverflow.com/questions/48234032/run-py-test-test-in-different-process
I tested it locally with `pip install pytest-forked pytest-xdist` and execute `pytest --forked python/tests/`, then all the tests can pass except for `test_manager.py` (global_history/test_manager_spark_dask.py). It will have four functions failed if adding '--forked'. I see the log and think the reason is: the four tests in this file depend on the results of other tests, and if using '--forked' the tests are all run independently, so they don't have the expected results and will fail.

I can come up with a solution as: not using `--forked` parameter for `test_manager.py` and using this parameter for other files. Or I can modify the tests in `test_manager.py` to make them independent and add `--forked` parameter to `pytest` here.

Reply:
Cool! Thanks for collecting the info! I think using --forked makes sense. Let's do that.